public/js/admin.js

```diff
@@ -605,7 +607,8 @@ var updateVisibility = function(qid) {
         function (isConfirm) {
             // User confirms the visiblity change
             if (isConfirm) {
-                question['visible'] = $('#checked-' + qid).is(':checked');
+                var visibilityStatus = $('#checked-' + qid).is(':checked');
```

Comment:
2 things
* why do we need a var here, why cant we do 
``` javascript
question['visible'] = $('#checked-' + qid).is(':checked');
```
* if we do need this variable, please change to `const`